light-base/src/json_rpc_service/background.rs

```diff
@@ -2553,11 +2585,44 @@ pub(super) async fn run<TPlat: PlatformRef>(
                         }));
                     }
 
+                    methods::MethodCall::transaction_v1_stop { operation_id } => {
+                        let exists = me
+                            .transactions_subscriptions
+                            .get(&*operation_id)
+                            .map_or(false, |sub| {
+                                matches!(sub.ty, TransactionWatchTy::NewApi { .. })
+                            });
+                        if exists {
+                            me.transactions_subscriptions.remove(&*operation_id);
+                            let _ = me
+                                .responses_tx
+                                .send(
+                                    methods::Response::transaction_v1_stop(())
+                                        .to_json_response(request_id_json),
+                                )
+                                .await;
+                        } else {
+                            let _ = me
+                                .responses_tx
+                                .send(parse::build_error_response(
+                                    request_id_json,
+                                    json_rpc::parse::ErrorResponse::ApplicationDefined(
+                                        -32800,
+                                        "Invalid operationId",
+                                    ),
```

Comment:
Note sure what the JSON-RPC error should be if `transaction_v1_stop` is called with an invalid id.

Reply:
I have returned `-32602` (invalid param error) in Substrate. We could probably clarify this in the spec